### PR TITLE
Resolver refactor alatham

### LINKED ISSUE #14: User - update resolvers
Add resolver for updating User model (see crate/resolvers.js for example) in api/src/modules/user/resolvers.js

### LINKED ISSUE #23: US - Style Summary
As a logged-in user
When I complete a style survey
I am given a summary of my style

### LINKED ISSUE #33: Develop algorithm for styleSummarization

### LINKED ISSUE #34: Refactor updateStyleSummary with helper functions and handling for edge case

### PR DESCRIPTION
**Pull request by alex-latham**
Refactors the user resolver for `updateStyleSummary` to:
* Handle an edge case where only one type of style is selected in the survey
* Utilize three helper methods: `tallyStyles`, `orderStyles`, and `generateStyleSummary`

Adds one more test for edge case (see first bullet).

Verify tests work by running `cd code/api && npm test` then tap `a` to run all API tests.

### CODE REVIEW

File: code/api/src/modules/user/mutations.test.js [hunk 1]
```diff
@@ -78,22 +78,22 @@ describe("user mutations", () => {
...
+      }`
+    }
+
+    var response = await request(server)
+      .post('/')
+      .send(data)
+      .expect(200)
+
+    expect(response.body.data.userStyleSummaryUpdate.styleSummary).toEqual("comfy")
+  })
```
Comment (reid-andrew) [inline code/api/src/modules/user/mutations.test.js:98]: Nice refactoring on this.

**Review by reid-andrew — APPROVED** [review]
